### PR TITLE
Error Reporting: 3rd Party schemas into their own folders

### PR DESCRIPTION
**Pull request by jerneju**

### CODE REVIEW

File: error_report/views.py [hunk 1]
```diff
@@ -23,6 +28,9 @@ def v1(request):
     widget_module, colon, lineno = widget_module.rpartition(":")
     module = request.POST.get("Module", "unknown")
     rel_path = _cleanup(os.path.join(widget_module, version, module))
+    for d in PATH_3RDPARTY.keys():
```
Comment (astaric) [inline error_report/views.py]: for d in PATH_3DPARTY: is enough

File: error_report/views.py [hunk 2]
```diff
@@ -23,6 +28,9 @@ def v1(request):
     widget_module, colon, lineno = widget_module.rpartition(":")
     module = request.POST.get("Module", "unknown")
     rel_path = _cleanup(os.path.join(widget_module, version, module))
```
Comment (astaric) [inline error_report/views.py]: I would refactor this into a function called get_report_path and call it here.

File: error_report/views.py (hunk 1 above)
Comment (jerneju) [reply error_report/views.py]: Fixed.

File: error_report/views.py (hunk 2 above)
Comment (jerneju) [reply error_report/views.py]: Fixed.

File: error_report/views.py [hunk 3]
```diff
@@ -12,18 +12,30 @@
...
+    "textable": "textable"
+}
+
+
 def _cleanup(path):
     return os.path.abspath(os.path.join('/', path))[1:]
 
 
+def get_report_path(rel_path):
+    for d in PATH_3RDPARTY:
```
Comment (astaric) [inline error_report/views.py]: Put all the path construction code here, add a docstring

**Pull request merged**